packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx

```diff
@@ -153,7 +153,7 @@ export const Hint: React.FC<HintProps> = ({ hintDisplayText }) => {
       // the TOO_CLOSE text, but for FACE_IDENTIFED, CANT_IDENTIFY, TOO_MANY
       // we are defaulting to the TOO_FAR text (for now). For MATCHED state,
       // we don't want to show any toasts.
-      return faceMatchState !== FaceMatchState.MATCHED ? (
+      return (
```

Comment:
Are we removing this condition because hint flickered too?

Reply:
Yep, the hint text is the text like `move back`, `move forward`, or `hold still` that shows above the progress bar. The progress bar and the hint text were both flickering.